hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/Target.java

```diff
@@ -38,7 +38,7 @@
     /**
      * Maximum length of key of controller attribute
      */
-    int CONTROLLER_ATTRIBUTE_KEY_SIZE = 32;
+    int CONTROLLER_ATTRIBUTE_KEY_SIZE = 128;
```

Comment:
check if Mgmt API and UI validate this

Reply:
There is no possibility to add/change controller attributes within Mgmt API/UI as they are coming from devices per DDI/DMF, and are additionally validated in `JpaControllerManagement`